Private/Set-AdditionalCAProperty.ps1

```diff
@@ -5,7 +5,7 @@
             Mandatory = $true,
             ValueFromPipeline = $true)]
         [array]$ADCSObjects,
-        [System.Management.Automation.PSCredential]$Credential
+        [PSCredential]$Credential
```

Comment:
Nice, this looks cleaner. Is the full type name implied? Any other benefits or reasons for shortening it?

Reply:
Uhhhh, you're asking me questions I don't have answers to. Might've been unintentional. Does it still work?

Reply:
Yes, [PSCredential] is fine. Woot.